python/python/res/job_queue/job_manager.py

```diff
@@ -136,6 +136,8 @@ def __init__(self, module_file="jobs.py", json_file="jobs.json", error_url=None,
         if log_url is None:
             self._log_url = error_url
         self._data_root = None
+        self.global_environment = None
+        self.global_update_path = None
```

Comment:
Here you can change the code slightly: Now the json file is *required*, and the module file should not be considered at all - i.e. the `_loadModule()`method can be completely removed.